workspaces/entity-feedback/plugins/entity-feedback-backend/src/service/router.test.ts

```diff
@@ -310,11 +320,25 @@ describe('createRouter', () => {
         userRef: 'user:default/mock',
         ...body,
       });
+      expect(mockNotificationService.send).toHaveBeenCalledWith({
+        recipients: {
+          type: 'entity',
+          entityRef: '',
```

Comment:
I think we can improve this test by modifying `sampleEntities[0]` above to add a owner relation so that this will test that the owner is correctly being set as the recipient